test/test_create_db.py

```diff
@@ -0,0 +1,77 @@
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import unittest
+import os
+import sqlite3
+import pandas as pd
+
+
+class TestDB(unittest.TestCase):
+
+    # create database and make sure it exists
+    def test_00_create_db_from_scratch(self):
+        conn = sqlite3.connect("test/FluxAccounting_test.db")
+
+        # create association table
+        conn.execute(
```

Comment:
shouldn't this be calling `create_db.py`?

Reply:
yes, you're right. I'm not sure why I was making the database from scratch... I can't remember. I just pushed a change so that the unit test file calls `create_db.py` with its own file path to the **test** directory.